ibm/conns/config.go

```diff
@@ -1707,15 +1707,19 @@ func (c *Config) ClientSession() (interface{}, error) {
 
 	// SCC Service
 	sccApiClientURL := scc.DefaultServiceURL
-	// Construct the service options.
-
-	sccApiClientOptions := &scc.SecurityAndComplianceCenterApiV3Options{
-		Authenticator: authenticator,
-		URL:           EnvFallBack([]string{"IBMCLOUD_SCC_API_ENDPOINT"}, sccApiClientURL),
+	_, ok := os.LookupEnv("IBMCLOUD_SCC_API_ENDPOINT")
```

Comment:
This is not acceptable solution where user needs to set th endpoint. Terraform should be able to construct the endpoint and intialize the client.
Terraform in intergarted with many other services like Scheamtics, Catalog, Projects. When they invoke SCC from these services they don't have flexible way to set this endpoint

Reply:
let's disregard this PR then.